src/store/search.js

```diff
@@ -346,7 +346,9 @@ const actions = {
       }
       commit(CLEAR_OTHER_MEDIA_TYPE_FILTERS, { searchType })
     }
-    await dispatch(UPDATE_QUERY_FROM_FILTERS, queryParams)
+    if (queryParams !== {}) {
```

Comment:
This python-ism won't work in JavaScript 😢  If we need to test for an empty object there are two ways we could do it. Either we could test `Object.keys(...).length` and check it for `0` (which I'm assuming has some caveats that I can't think of, it seems like it should work fine) or we could use the `lodash.isEmpty` function.